.gitignore

```diff
@@ -27,11 +27,15 @@ out.txt
 
 # Pipenv and venv
 Pipfile.lock
-pyproject.toml
 venv
 
 # Test files
 pytest_reports/
+.coverage
 
 # Documentation
 _docs
+
+# Cython compilation
+*.cpp
```

Comment:
ignoring cpp files broadly is a bit overkill. I would place this on cython files only